spras/meo.py

```diff
@@ -167,4 +179,7 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
         # TODO what should be the edge rank?
         # Would need to load the paths output file to rank edges correctly
         df.insert(5, 'Rank', 1)  # Add a constant rank of 1
-        df.to_csv(standardized_pathway_file, columns=['Source', 'Target', 'Rank'], header=False, index=False, sep='\t')
+
+        df = readd_direction_col_mixed(df, "Type", "pd", "pp")
+
+        df.to_csv(standardized_pathway_file, columns=['Source', 'Target', 'Rank', "Direction"], header=False, index=False, sep='\t')
```

Comment:
Eventually we want to start writing the header row as part of the standard pathway output format. Is that part of this pull request or a separate change?